src/main/java/com/capgemini/molvenoresort/services/BookingService.java

```diff
@@ -0,0 +1,58 @@
+package com.capgemini.molvenoresort.services;
+
+import com.capgemini.molvenoresort.models.Booking;
+import com.capgemini.molvenoresort.repositories.bookingRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Optional;
+
+@Service
+public class BookingService {
+
+    @Autowired
+    private bookingRepository repository;
+
+
+    public Iterable<Booking>findAll() {
+        return this.repository.findAll();
+    }
+
+    public ResponseEntity<Booking> findById(long id) {
+        Optional<Booking> temp = repository.findById(id);
+        return temp.map(ResponseEntity::ok).orElseGet(() -> ResponseEntity.notFound().build());
+    }
+
+    public ResponseEntity deleteById(long id) {
+        repository.deleteById(id);
+        return ResponseEntity.noContent().build();
+    }
+
+    public ResponseEntity doesBookingExist(long id) {
+        return ResponseEntity.ok(repository.findById(id).isPresent());
+    }
+
+
+    public ResponseEntity<?> addBooking(Booking booking) {
```

Comment:
Al een comment in controller geplaatst. de checkAge returned de goede waarde, maar de responseEntity die ontvangen wordt in controller klopt niet helemaal